### ISSUE TITLE
报错求助

### ISSUE THREAD

**Issue by SJseventeen** [anliyuan/Ultralight-Digital-Human#140]
OpenCV: FFMPEG: tag 0x47504a4d/'MJPG' is not supported with codec id 7 and format 'mp4 / MP4 (MPEG-4 Part 14)'
OpenCV: FFMPEG: fallback to use tag 0x7634706d/'mp4v'
D:\anaconda\envs\dh\lib\site-packages\torch\nn\functional.py:1967: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
请问大佬们，这三个报错应该如何去处理，有稍详细的步骤吗，万分感谢

**Comment by anliyuan** [comment]
warning不用管
